README.md

```diff
@@ -112,6 +112,8 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
 3. There are a few ways you can locally install the `ilab` CLI. Select your preferred installation method from the following instructions. You can then install `ilab` and activate your `venv` environment.
 
+   > **NOTE**: ⏳ The `python3` CLI shown in the following steps is the Python that you installed in the above step. The command can be `python3.11` or `python3.10` instead of `python3`. You can check Python's version by `python3 -V`.
```

Comment:
```suggestion
   > **NOTE**: ⏳ The `python3` binary shown in the following steps is the Python version that you installed in the above step. The command can be `python3.11` or `python3.10` instead of `python3`. You can check Python's version by `python3 -V`.
```

Reply:
Markdown linting also needs to be fixed

Reply:
All right. Let me fix those issues.

Reply:
Wait. The NOTE in the quote block is broken. Let me fix it.

https://github.com/junaruga/instructlab/blob/wip/readme-python-cli/README.md#-installing-ilab

Reply:
> Wait. The NOTE in the quote block is broken. Let me fix it.
> 
> https://github.com/junaruga/instructlab/blob/wip/readme-python-cli/README.md#-installing-ilab

OK. I fixed the issue on the latest rebase.

Reply:
> Thanks, you can also run it locally via `make markdown-lint`

Thanks for the info. I believe that the command is `make md-lint` seeing the `Makefile`. And I confirmed the command passed on my local environment.

Reply:
Oh yes you are right!